### PR TITLE
Quality audit - Fixing additional data help urls

### PR DESCRIPTION
**Pull request by alex-vazquez**
---
### Purpose of this PR
https://jira.unity3d.com/browse/XPIPELINE-194
https://jira.unity3d.com/browse/XPIPELINE-201

---
### Testing status
- Check multiple links on HDRP/ URP
- Check that de URP additional datas have a correct documentation page

---
### Comments to reviewers
I did a clean up on the Documentation clases, and a tiny refactor.

### CODE REVIEW

File: com.unity.render-pipelines.core/Runtime/Volume/Volume.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@ namespace UnityEngine.Rendering
     /// <summary>
     /// A generic Volume component holding a <see cref="VolumeProfile"/>.
     /// </summary>
-    [HDRPHelpURLAttribute("Volumes")]
+    [CoreRPHelpURL("Volumes", "com.unity.render-pipelines.high-definition")]
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.core/Runtime/Volume/Volume.cs:8]: wait, why does this one point to HDRP?

File: com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ namespace UnityEngine.Rendering
     /// <summary>
     /// An Asset which holds a set of settings to use with a <see cref="Volume"/>.
     /// </summary>
-    [HDRPHelpURLAttribute("Volume-Profile")]
+    [CoreRPHelpURL("Volume-Profile", "com.unity.render-pipelines.high-definition")]
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs:10]: should we move the doc to Core package instead?

File: com.unity.render-pipelines.universal/Runtime/Documentation.cs [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+        /// The name of the package
+        /// </summary>
+        public const string packageName = "com.unity.render-pipelines.universal";
+
+        /// <summary>
+        /// Generates a help url for the given package and page name
+        /// </summary>
+        /// <param name="pageName">The page name</param>
+        /// <returns>The full url page</returns>
+        public static string GetPageLink(string pageName) => GetPageLink("com.unity.render-pipelines.universal", pageName);
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.universal/Runtime/Documentation.cs:26]: should it be GetPageLink(packageName, pageName) ?

**Review by jenniferd-unity — CHANGES_REQUESTED** [review]

File: com.unity.render-pipelines.universal/Runtime/Documentation.cs (hunk 3 above)
Comment (alex-vazquez) [reply com.unity.render-pipelines.universal/Runtime/Documentation.cs]: yeah, thanks! Last refactor that I didn't notice :)

File: com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs (hunk 2 above)
Comment (alex-vazquez) [reply com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs:10]: The docs team should do a pass on this :) @Vic-Cooper , @oleks-k, there are pages from hdrp that are referenced from core. Let us know when you modify them and we will remove this string.

File: com.unity.render-pipelines.core/Runtime/Volume/Volume.cs (hunk 1 above)
Comment (alex-vazquez) [reply com.unity.render-pipelines.core/Runtime/Volume/Volume.cs:8]: yeah they moved the code but not the docs, at leas know is just an string that we can easily remove.

**Review by jenniferd-unity — APPROVED** [review]
Approving for now. The update from Docs should probably follow

File: com.unity.render-pipelines.core/Runtime/Volume/Volume.cs (hunk 1 above)
Comment (RSlysz) [reply com.unity.render-pipelines.core/Runtime/Volume/Volume.cs:8]: If I remember well, there is history here.
Beyond memories, there was a time where no one remember Volume being in LightWeightRP (old name for UniversalRP).
Then a challenger appeared that change the face of RP world. Volume was introduced in this LWRP/URP too.
Though, both RP have really different components and both had there own documentation link.

Now:
Is the link working when in URP or does it redirect to unwanted documentation?

**Review by RSlysz — APPROVED** [review]
lgtm.

**Review by ellioman — APPROVED** [review]

**Review by erikabar — APPROVED** [review]
Checked Documentation, Changelog, Licence links, also checked help button on UniversalAdditionalCameraData and Volume - everything is working as expected.

**Pull request merged**